testprojects/src/python/python_distribution/ctypes_with_extra_compiler_flags/some_more_math.hpp

```diff
@@ -0,0 +1,22 @@
+#ifndef __SOME_MORE_MATH_HPP__
+#define __SOME_MORE_MATH_HPP__
+
+#ifdef NDEBUG
+#define assert(condition) ((void)0)
+#else
+#define assert(condition)
+#endif
+
+#ifdef _GLIBCXX_USE_CXX11_ABI
```

Comment:
I would prefer if this example used a flag that wasn't tied to other behavior, although this is fine. All that would mean here would be to use something like `ASDF` instead of `_GLIBCXX_USE_CXX11_ABI`, since we're not testing anything specific to that flag here it looks like?